src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/AssessmentTransformer.kt

```diff
@@ -56,9 +63,28 @@ class AssessmentTransformer(
       rejectionRationale = jpa.rejectionRationale,
       status = getStatus(jpa)
     )
+
     else -> throw RuntimeException("Unsupported Application type when transforming Assessment: ${jpa.application::class.qualifiedName}")
   }
 
+  fun transformDomainToApiSummary(ase: DomainAssessmentSummary, offenderDetailSummary: OffenderDetailSummary, inmateDetail: InmateDetail): AssessmentSummary =
+    AssessmentSummary(
+      type = when (ase.type) {
+        "approved-premises" -> AssessmentSummary.Type.cAS1
+        else -> AssessmentSummary.Type.cAS3
```

Comment:
I'd maybe go for something slightly more gaurded like:

```
type = when (ase.type) {
        "approved-premises" -> AssessmentSummary.Type.cAS1
        "temporary-accommodation" -> AssessmentSummary.Type.cAS3
        else -> throw RuntimeException("Unsupported type: ${ase.type}")
}
```

Reply:
Done